### DISCUSSION TITLE
Is there a way to force the language for spellchecking with the API ?

### DISCUSSION THREAD

**Discussion by erwandf** [ONLYOFFICE/DocumentServer#2755] [Q&A]
Hi,
I'm using onlyoffice in a multilingual environment, and the spellchecking is always set on English.
I'd like to configure the language for our users depending on their locale.
Is there a way to do this, using the API ?

thanks and regards,

**Comment by Rita-Bubnova** [comment]
@regent112, Take a look, please.
